### ISSUE TITLE
This game is not good

### ISSUE THREAD

**Issue by Exrly** [1j01/slugg#1]
This game currently lacks coheivsible mechanics or storylines. There are too few graphics, and not enough code. All code is messy and graphics are bad. In game u take on role of man with really bad superpowers: u can run slightly slower than trains and jump ok, and get hit with trains as many times as u want. There is little to be said of the character, which is a total mary-sue whose only personality trait is having a robot arm and getting hit with many trains. You can also slide now but it looks weird.
![clack](https://cloud.githubusercontent.com/assets/16873188/12542806/07b7a74e-c2f6-11e5-89b7-70528a2620fb.png)

Basically, skyrim with trains, 0/10 GOTY

**Comment by 1j01** [comment]
Wow, rude. First of all, how dare you relate this to skyrim. There are no sticky bun related memes in this title let alone DARGONS or fusrodah. Second of all, good points excellent critique much appreciated kthxbai

**Comment by Exrly** [comment]
frist of all how **dare** yo u

Seconds: if there is not DRAGONS then what is smoke in attached picture from?? What is blurry shape? Possibly dragons. I demand apology and for you to admit to ignoring DRAGONS. Thnaxks for speedy response good game development across the board.

**Comment by 1j01** [comment]
the smoke is obviously _from_ a dragon, but there are no dragons _in the game_

what is this, [some kind of](https://vimeo.com/144006470)

**Comment by 1j01** [comment]
i think [k](https://s-media-cache-ak0.pinimg.com/originals/30/66/41/306641a43a8a47b148e9d032dce0b32a.jpg)not
